src/main/resources/static/common/js/item.js

```diff
@@ -35,24 +35,36 @@ app.controller('itemDetailController', function($rootScope, $http, $location, $f
 			var item = result.data.data;
 			itemDetail.item = item;
 			itemDetail.showItemDetail = true;
-			
-			$http({
-				method: 'GET',
-				url: '/proxy/v1/assets/categories/' + item.categoryId,
-				params: {categoryId: item.categoryId},
-			}).then(function(result) {
-				data = result.data.data;
-				itemDetail.categoryName = data.name;
-			}).catch(function(result) {
-				console.info(result);
-				displayLoadError(result,$rootScope,$filter,$http,true,'categories');
-			});
-		}).catch(function(result) {
-			console.info(result);
-			itemDetail.itemDetailError = true;
-			displayLoadError(result,$rootScope,$filter,$http,false,'items');
-		});
-	}, 500);
+
+            let success = (result) => {
+                itemDetail.categoryName = result.name;
+            }
+            let error = (result) => {
```

Comment:
Add a parameter to get endpoint type, and pass it into displayLoadError(result, $rootScope, $filter, $http, true, **endpointTypeHere**);.